report_viewer/settings/base.py

```diff
@@ -41,7 +41,7 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.getenv("SECRET_KEY")
+SECRET_KEY = os.getenv("SECRET_KEY", "")
```

Comment:
This is dangerous as this could lead to a service on PaaS being deployed without a secret key, and it would not be immediately obvious if the secret key is missing. 

If you need a default value, could it be None or raise an exception if the string is empty?

Reply:
This change is no longer needed since you just removed the use of `SECRET_KEY` from `report_viewer/apps/viewer/middleware/report_views_middleware.py`. Pylance was complaining about the latter.

d8ac8362b31ae46bd96ee602ea9adea014a0bb4b